src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdCaseUpdaterTest.java

```diff
@@ -311,10 +308,7 @@ void updateCase_should_handle_feign_exception() {
             .willReturn(noWarningsUpdateResponse);
         initMockData();
         prepareMockForSubmissionEventForCaseWorker().willThrow(
-            new FeignException.BadRequest(
-                "Msg",
-                mock(Request.class),
-                "Body".getBytes())
+            new RuntimeException("Service response: Body")
```

Comment:
why RuntimeException?

Reply:
Because previously we were mocking `CoreCaseDataApi` which throws `FeignException`, but now we are mocking here `CcdApi` which catches `FeignException` and converts it into `RuntimeException`.

Reply:
better to throw specific exception